src/ripple/protocol/STBase.h

```diff
@@ -126,9 +127,6 @@ class STBase
     SField::ref
     getFName() const;
 
-    std::unique_ptr<STBase>
-    clone() const;
```

Comment:
The reason I was nattering about `clone` is that all of the base classes used to get this (non-virtual) `clone` interface and implementation.  Internally that `clone` implementation called the private virtual `duplicate` methods.  You got rid of the indirection from `clone` to `duplicate`.  That's great.  Anyway, that's why I've been saying that the name changed from `clone` to `duplicate`.  You're correct, the old `duplicate` methods were there before.  But the public access to those methods was through a `clone` method that's now gone.

I'm fine with your decision to postpone the rename.  I just thought I'd try to explain myself a bit more clearly.